explanation/intro-to/crypto-libraries.md

```diff
@@ -21,70 +22,64 @@ Among the most popular and widely used libraries and frameworks, we have:
+The most common options are:
 
-First, let's take a look at how an application could use crypto.
+### Follow dynamic links
 
-### Dynamic linking
-
-This is the most common way, and very easy to spot via package dependencies and helper tools. This is discussed later in this page.
+This is the most common way, and very easy to spot via package dependencies and helper tools. A detailed example of this approach to determining crypto is discussed later in this page.
  
-### Static linking
+### Check static linking
 
-This is harder, as there is no dependency information in the binary package, and this usually requires inspection of the source package to see Build Dependencies. An example is shown later in this page.
+This is harder, as there is no dependency information in the binary package, and this usually requires inspection of the source package to see build dependencies. A detailed example of this approach is shown later in this page.
 
-### Plugins
+### Look at the plugins
 
 The main binary of an application can not depend directly on a crypto library, but it could load dynamic plugins which do. Usually these would be packaged separately, and then we fall under the dynamic or static linking cases above. Note that via such a plugin mechanism, an application could depend on multiple external cryptographic libraries.
 
-### Execution of external binary
+### Examine the execution of external binaries
 
 The application could just plain call external binaries at runtime for its cryptographic operations, like calling out to `openssl` or `gnupg` to encrypt/decrypt data. This will hopefully be expressed in the dependencies of the package. If it's not, then it's a bug that should be reported.
 
-### Indirect usage
+### Indirect use of libraries or executables
 
 The application could be using a third party library or executable which in turn could fall into any of the above categories.
 
-## Identify the crypto libraries used by an application
-
-Here are some tips that can help identifying the crypto libraries used by an application that is installed on an Ubuntu system:
-
-### Documentation
+### Check documentation more carefully
 
-Read the application documentation. It might have crypto options directly in its own configuration files, or point at specific crypto configuration files installed on the system. This may also clarify if the application even uses external crypto libraries, or if it has its own implementation.
+Look at the application documentation again. It might have crypto options directly in its own configuration files, or point at specific crypto configuration files installed on the system. This may also provide you with assistance if the application uses external crypto libraries or whether it has its own implementation of crypto.
```

Comment:
```suggestion
Read the application documentation again. It might have crypto options directly in its own configuration files, or point at specific crypto configuration files installed on the system. This may also clarify whether the application uses external crypto libraries or it has its own implementation of crypto.
```